packages/wdio-logger/src/node.ts

```diff
@@ -131,21 +131,22 @@ export default function getLogger (name) {
  * Calling this prevents part of the logs in the very env to be lost.
  */
 getLogger.waitForBuffer = async () => new Promise(resolve => {
+    // @ts-ignore
```

Comment:
is it really required to have this one?

Reply:
yeah, I couldn't find a `writableBuffer` property anywhere in node. It might be a private property and not suppose to be used but we should clean this in a separate PR.

Reply:
tried to overcome it myself, and didn't find any solution. Makes sense!